### ISSUE TITLE
Personalized templates

### ISSUE THREAD

**Issue by folago** [k1LoW/tbls#257]
The embedded templates are great to get you started with `tbls doc` but depending on the DB or the project it might fall short in some aspects.

It would be very helpful to be possible to specify a personalized template at execution time and inside the tbls.yml config file.

E.g.:
```
# tbls.yml
er:
  format: svg
  comment: true
  distance: 2
  font: M+
  template: "path/to/dot/template"
```

```sh
$tbls out -t plantuml -template "path/to/puml/template" -o schema.puml
```

**Comment by k1LoW** [comment]
Hi @folago!
Thank you for your suggestion and https://github.com/k1LoW/tbls/pull/258.

I like the idea of being able to plug in templates externally.

However, templates are not  a single file.

In the case of `tbls doc`, there are templates in [`.dot` files](https://github.com/k1LoW/tbls/tree/master/output/dot/templates) as well as in [`.md` files](https://github.com/k1LoW/tbls/tree/master/output/md/templates).

I have yet to come up with a good design for the above.

**Comment by folago** [comment]
I see, what would you think then of adding another section `templates` to the config file?

Something like this:

```yaml

templates:
  md:
    table: path/to/schema.tmpl
  dot:
    schema: path/to/schema.dot.tmpl
    table: path/to/table.dot.tmpl
  puml:
    etc....  
```

So it would be possible to override the templates specified in the config, and the rest would be left with the default.

**Comment by k1LoW** [comment]
Looks Good!

**Comment by folago** [comment]
NIce! I'll prepare a PR :grin:

**Comment by folago** [comment]
PR #258 updated :slightly_smiling_face:

**Comment by k1LoW** [comment]
Thank you @folago ! 

Released as v1.44.0.
